specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,910 @@
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of Namespaces."
+        }
+      },
+      "description": "The response of a list Providers operation."
+    },
+    "QuantumWorkspace": {
+      "type": "object",
+      "description": "The resource proxy definition object for quantum workspace.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The unique id for this workspace.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of this workspace.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of this workspace.",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "description": "The location (region) of this workspace."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the properties. Define quantum space's specific properties.",
+          "$ref": "#/definitions/WorkspaceResourceProperties"
+        },
+        "tags": {
+          "type": "object",
+          "description": "The tags associated with this workspace."
+        },
+        "identity": {
+          "type": "object",
+          "description": "Managed Identity information",
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "Type of identity. Supported: SystemAssigned or None (case insensitive)"
+            },
+            "principalId": {
+              "type": "string",
+              "description": "objectId of the managed identity,"
+            },
+            "tenantId": {
+              "type": "string",
+              "description": "tenantId where the managed identity lives."
+            }
+          }
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "WorkspaceResourceProperties": {
+      "description": "Properties of a Workspace",
+      "properties": {
+        "providers": {
+          "type": "array",
+          "description": "List of Providers selected for this Workspace",
+          "items": {
+            "$ref": "#/definitions/Provider"
+          }
+        },
+        "usable": {
+          "type": "string",
+          "enum": [
+            "Yes",
+            "No",
+            "Partial"
+          ],
+          "x-ms-enum": {
+            "name": "UsableStatus",
+            "modelAsString": true
+          },
+          "description": "Whether the current workspace is ready to accept Jobs."
+        },
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "ProviderLaunching",
+            "ProviderUpdating",
+            "ProviderDeleting",
+            "ProviderProvisioning",
+            "Deleted",
```

Comment:
>Deleted [](start = 13, length = 7)

Deleted is not a valid terminal provisioning state. The only supported terminal states are Succeeded, Canceled, and Failed.

Reply:
Removed Deleted.